geode-core/src/main/java/org/apache/geode/internal/logging/LogWriterImpl.java

```diff
@@ -1030,21 +1097,21 @@ public void run() {
             return;
           }
           if (!done.get() && targetThread.isAlive()) {
-            StringBuffer sb = new StringBuffer(500);
+            StringBuilder sb = new StringBuilder(500);
             if (toStdout) {
               sb.append("[trace ").append(getTimeStamp()).append("] ");
             }
             StackTraceElement[] els = targetThread.getStackTrace();
-            sb.append("Stack trace for '").append(targetThread.toString()).append("'\n");
+            sb.append("Stack trace for '").append(targetThread).append("'\n");
```

Comment:
What do you want to replace the hardcoded newlines with? Without them the stack trace will be all on one line.

Reply:
Sorry, my original comment was too terse. My intent was as @sboorlagadda said. All part of the effort to remove platform dependencies like pathname separators, end of line conventions, classpath separators, etc.

Reply:
I think we should have change every \n in the code base to be System.getProperty("line.separator") in one PR, but I'll go ahead and change this class.

Reply:
@pdxrunner @sboorlagadda I'm actually not sure how to handle some of this logic:
```java
      if (endChar == '\n') {
        // trim off the \n since println will do it for us
        wordEnd--;
        if (wordEnd > 0 && target.charAt(wordEnd - 1) == '\r') {
          wordEnd--;
        }
```
I can't change that to System.getProperty("line.separator")

How about I merge this in and then I file a ticket to change all \n to be platform independent and I pair with one or both of you to fix everything related to \n?

Reply:
Sounds OK.